src/App.js

```diff
@@ -1,76 +1,77 @@
-import React, { useEffect, useState } from "react";
-import "./App.css";
-import logo from "./logo.png";
+import { useEffect, useState } from "react";
+import React from 'react'
+import './App.css';
+import Search from "./Search";
 import Box from "./Components/Box";
+import logo from './mlh-prep.png'
 
 function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
-  const [generic, setGeneric] = useState("app");
-  const [city, setCity] = useState(null);
+  const [city, setCity] = useState("New York City")
   const [results, setResults] = useState(null);
-
+  const [generic, setGeneric]=useState("app");
+  const [notfound,setFlag]=useState(false);
   useEffect(() => {
-    window.navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
Ahh, i think this isn't the duplicate code, it's for getting user's location, it should be there.

Reply:
Btw you mistakenly added Autocomplete SearchBar code within this PR.  😅